scripts/ibu10m/collection.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import os
+from glob import glob
+
+import pystac
+from pystac.extensions.item_assets import AssetDefinition, ItemAssetsExtension
+
+from .constants import (
+    CLMS_LICENSE,
+    COLLECTION_DESCRIPTION,
+    COLLECTION_EXTENT,
+    COLLECTION_ID,
+    COLLECTION_KEYWORDS,
+    COLLECTION_SUMMARIES,
+    COLLECTION_TITLE,
+    IBU10M_HOST_AND_LICENSOR,
+    STAC_DIR,
+    TITLE_MAP,
+    WORKING_DIR,
+)
+
+
+def create_collection(item_list: list[str]) -> pystac.Collection:
```

Comment:
I will break this function into small functions. For example, [create_core_collection](https://github.com/eea/clms-stac/blob/develop/scripts/uabh/collection.py#L76), [add_summaries_to_collection](https://github.com/eea/clms-stac/blob/develop/scripts/uabh/collection.py#L87), [add_item_assets_to_collection](https://github.com/eea/clms-stac/blob/develop/scripts/uabh/collection.py#L92), [add_links_to_collection](https://github.com/eea/clms-stac/blob/develop/scripts/uabh/collection.py#L97), and [add_items_to_collection](https://github.com/eea/clms-stac/blob/develop/scripts/uabh/collection.py#L102)